dependencies/lib/perl/Transcript.pm

```diff
@@ -68,6 +68,8 @@ sub get_feature {
 
 	return 'intergenic' if($bp > $self->end && $strand eq '+');
 	return 'intergenic' if($bp < $self->start && $strand eq '-');
+	return 'intergenic' if($bp < $self->start && $strand eq '+');#Tian
+        return 'intergenic' if($bp > $self->end && $strand eq '-');#Tian
```

Comment:
Couldn't we just remove the strand check, rather than duplicating the check for each strand?